server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -825,7 +883,6 @@ private IndexMetadata getIndexMetadata(String clusterName, String clusterUUID, U
      * @return {@link IndexMetadata}
      */
     public ClusterState getLatestClusterState(String clusterName, String clusterUUID) {
-        start();
```

Comment:
why was it here before?

Reply:
This is already called in Node.java. I also checked with @soosinha  on removing this and he also acknowledged that this should have been removed earlier and was missed.